bears/general/IndentationBear.py

```diff
@@ -222,7 +223,8 @@ def get_indent_levels(self,
                 if _range.start.line == line + 1:
                     next_indent += 1
 
-                first_ch = file[line].lstrip()[0] if file[line].lstrip() else ""
+                first_ch = file[line].lstrip()[0] if file[line].lstrip() \
+                    else ""
```

Comment:
@Ana06 @Adrianzatreanu This is one way. 
```python
first_ch = (file[line].lstrip()[0]
            if file[line].lstrip()
            else "")
```
Note: For demo it has been shown in 3 lines. You may go with two lines only, keeping else in second line. Personally, I like 3 lines split style. :)